pkg/gardenlet/controller/seed/seed/nginxingress.go

```diff
@@ -54,11 +54,16 @@ func defaultNginxIngress(
 	}
 
 	values := nginxingress.Values{
+		ClusterType:             component.ClusterTypeSeed,
+		TargetNamespace:         gardenNamespaceName,
+		IngressClass:            v1beta1constants.SeedNginxIngressClass,
 		ImageController:         imageController.String(),
 		ImageDefaultBackend:     imageDefaultBackend.String(),
-		IngressClass:            ingressClass,
+		PriorityClassName:       v1beta1constants.PriorityClassNameSeedSystem600,
 		ConfigData:              config,
 		LoadBalancerAnnotations: loadBalancerAnnotations,
+		PSPDisabled:             true,
```

Comment:
Rather call it `PSPEnabled` (most fields are called "enabled" instead of "disabled")

Reply:
In all components currently it's `PSPDisabled`, it wouldn't be nice to change it only here.